Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -2434,6 +2442,11 @@ ModelExperimental.prototype.applyStyle = function (style) {
     const featureTable = this.featureTables[this.featureTableId];
     featureTable.applyStyle(style);
   } else {
+    // check if the feature table id is undefined, or
+    // if the table doesn't exist
+    // if so, use empty feature table?
+
+    // empty feature table.applyStyle
```

Comment:
since we opted not to add an empty feature table, I'd update/remove this comment